src/lib/components/01-elements/img/img.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactNode } from 'react';
+import Typography from '../typography/typography';
+import { cn } from '../../../utils/tailwind-merge';
+
+type ImgProps = {
+  src: string;
+  description: string;
+  bgColor?: 'dark' | 'light';
+  renderedImageComponent?: ReactNode;
+};
+
+export default function Img({
+  src,
+  description,
+  bgColor = 'light',
+  renderedImageComponent,
+}: ImgProps) {
+  return renderedImageComponent ? (
```

Comment:
Component updated. Just wanted to point out that the end component is very different from what was asked in the ticket. It would be nice to list the use cases in over there…